### PR TITLE
Cherry-pick customizations

### PR DESCRIPTION
**Pull request by gronke**
Cherry-pick customizations prior to 2.0.

### CODE REVIEW

File: source/snippets/offerte/nl/teamandreporting.xml [hunk 1]
```diff
@@ -40,6 +40,20 @@
...
+                        <li>
+                                <a
+                                        href="https://www.radicallyopensecurity.com/our-portfolio/"
+                                >
+                                        radicallyopensecurity.com/our-portfolio
+                                </a>
+                        </li>
+                </ul>
+
+>>>>>>> 3ebdfd6 (fix(snippet): point to website for portfolio)
```
Comment (gronke) [inline source/snippets/offerte/nl/teamandreporting.xml:56]: Oops 👀

File: xslt/findings2jira.xsl [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
```
Comment (syrosh) [inline xslt/findings2jira.xsl:1]: This file is no longer needed. A JIRA field is included in the main export

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+stages:
```
Comment (syrosh) [inline .gitlab-ci.yml:1]: I'm fine with this, but I thought @PeterMosmans wanted this in a separate repo

**Review by syrosh — CHANGES_REQUESTED** [review]
findings2jira can go

@PeterMosmans can you please review, specifically the gitlab-ci.yml file?

File: xslt/findings2jira.xsl (hunk 2 above)
Comment (PeterMosmans) [reply xslt/findings2jira.xsl:1]: I agree with @syrosh : @gronke , could you please remove this file from this PR? Thanks

File: README.md [hunk 4]
```diff
@@ -52,6 +52,12 @@ text editor like _JEdit_, to a full IDE- for editing of course ;). Preferably
 something that can check XML file validity. To view the resulting PDF files a
 PDF viewer is necessary.
 
+### Building PDF's
```
Comment (PeterMosmans) [inline README.md:55]: This looks like a duplicate one. I prefer not to add this, as the toolchain repository will be 'revamped'

File: README.md [hunk 5]
```diff
@@ -76,6 +76,27 @@ To compile a CSV run:
 make export-csv
 ```
 
+### Exporting CSV
```
Comment (PeterMosmans) [inline README.md:79]: This is specific for GitLab, and should IMHO be part of the toolchain - not this repo. However, as this file already contains build instructions there's not that much downside

File: xslt/findings2jira.xsl (hunk 2 above)
Comment (gronke) [reply xslt/findings2jira.xsl]: Commit removed.

File: .gitlab-ci.yml (hunk 3 above)
Comment (gronke) [reply .gitlab-ci.yml]: CI/CD config has been removed.

File: README.md (hunk 4 above)
Comment (PeterMosmans) [reply README.md:55]: Hey @gronke - it looks like this is the only outstanding issue for this PR. Could you please ensure that there is only one header? IMHO the correct spelling is `Building PDFs`

**Pull request merged**